frontend/packages/topology/src/components/quick-search/QuickSearchList.scss

```diff
@@ -0,0 +1,29 @@
+.odc-quick-search-list {
+  &__item {
+    &--highlight {
+      background: var(--pf-global--active-color--200);
+    }
+    &:hover {
+      background: #cbe9f9;
```

Comment:
Hmmm, no PF colour?

Reply:
No on hover I wanted a color lighter than what appears when we select an item but didnt find it in pf

Reply:
Hmmm... @bgliwa01 for Monday, can you provide some feedback on the chosen colour?

Reply:
@divyanshiGupta can we follow the same hover treatment that is used in the topology list view? you might be doing this - it looks very similar.

Reply:
@divyanshiGupta see if the colours match, report that status. If they do, then we are _good_ here... otherwise we should align to Topology (and provide a screenshot for Brigid to verify).

Reply:
List view is using the same color on hover as I have used for highlighting the item when it is selected.
![Screenshot from 2020-11-30 20-38-59](https://user-images.githubusercontent.com/20724543/100626050-4ea60900-334b-11eb-8ebd-b8c9b601e086.png)